src/ontology/cl-edit.owl

```diff
@@ -30820,6 +30834,119 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32555229") Annotation(ob
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033020 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033020 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033020 "2023-03-16T13:03:13Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:18931053") oboInOwl:hasExactSynonym obo:CL_4033020 "tracheal submucosal gland mucous cell")
+AnnotationAssertion(rdfs:label obo:CL_4033020 "mucus secreting cell of trachea gland")
+EquivalentClasses(obo:CL_4033020 ObjectIntersectionOf(obo:CL_0000319 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005203)))
+SubClassOf(obo:CL_4033020 obo:CL_0000319)
+
+# Class: obo:CL_4033021 (myoepithelial cell of trachea gland)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29656943") Annotation(oboInOwl:hasDbXref "PMID:31651362") obo:IAO_0000115 obo:CL_4033021 "A myoepithelial cell that is part of a submucosal gland of the trachea.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033021 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033021 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033021 "2023-03-16T13:36:43Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4033021 "myoepithelial cell of trachea gland")
+EquivalentClasses(obo:CL_4033021 ObjectIntersectionOf(obo:CL_0000185 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005203)))
+
+# Class: obo:CL_4033022 (mucus secreting cell of bronchus submucosal gland)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19965983") obo:IAO_0000115 obo:CL_4033022 "A mucus secreting cell of a submucosal gland of the bronchus.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033022 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033022 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033022 "2023-03-16T15:12:52Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:5487122") oboInOwl:hasExactSynonym obo:CL_4033022 "bronchial gland mucous cell")
+AnnotationAssertion(rdfs:label obo:CL_4033022 "mucus secreting cell of bronchus submucosal gland")
+EquivalentClasses(obo:CL_4033022 ObjectIntersectionOf(obo:CL_0000319 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_8410043)))
+
+# Class: obo:CL_4033023 (airway submucosal gland collecting duct epithelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25648514") Annotation(oboInOwl:hasDbXref "PMID:29656943") Annotation(oboInOwl:hasDbXref "PMID:31651362") obo:IAO_0000115 obo:CL_4033023 "An epithelial cell that is part of a collecting duct of an airway submucosal gland.")
```

Comment:
After cursory review, did not find this cell type mentioned in PMID:31651362.

Reply:
I've removed this reference from all cell terms that is was requested.